components/elm/docs/tech-guide/shortwave_radiation.md

```diff
@@ -0,0 +1,807 @@
+solutions for the upward and downward diffuse radiation
+using the two-stream approximation are given. Third, the
+analytical solutions are used for the single-layer canopy
+configuration to compute the radiation absorbed by the
+canopy and ground and the radiation absorbed by the
+canopy for sunlit and shaded leaves. Fourth, the
+treatment of surface albedo is described. Fifth, the
+equations describing the total amount of solar radiation
+are presented.
+
+## Two-Stream Approximation Radiative Transfer Model
+
+At any level within the canopy, the upward/downward
+direct/diffuse radiation is transmitted and intercepted
+by the canopy. The intercepted radiation is scattered
+backward, scattered forward, and absorbed by the canopy.
+The diffuse radiative fluxes via the two-stream
+approximation are given as
+
+$$
+\begin{eqnarray}
+- \dfrac{dI^{\uparrow}}{d(L+S)} + [ 1 - (1 - \beta)\omega ]
+\dfrac{1}{\bar{\mu}}I^{\uparrow} - \omega \beta \dfrac{1}{\bar{\mu}}
+I^{\downarrow} = \omega K_b \beta_0 e^{-K(L+S)}
+\label{eqn:dif_up} \\[0.5em]
+ \dfrac{dI^{\downarrow}}{d(L+S)} + [ 1 - (1 - \beta)\omega ]
+\dfrac{1}{\bar{\mu}} I^{\downarrow} - \omega \beta \dfrac{1}{\bar{\mu}}
+I^{\uparrow} = \omega K_b (1 - \beta_0) e^{-K(L+S)}
+\label{eqn:dif_dn}
+\end{eqnarray}
+$$
+
+where $I^{\uparrow}$ is the upward diffuse radiative flux
+per unit incident flux, $I^{\downarrow}$ is the downward
+diffuse radiative flux per unit incident flux,
+$K_b \left( = G(\mu)/\mu \right)$ is the optical depth of
+direct radiation per unit leaf and stem area, $G(\mu)$ is
+the relative projected area of leaf and stem in the
+direction of $\cos^{-1}(\mu)$, $\mu$ is the solar zenith
+angle, $\bar{\mu} \left( = 1/K_d \right)$ is the inverse
+of the optical depth of diffuse radiation per unit leaf
+and stem area, $\omega$ is the scattering coefficient,
+$\beta$ is the upscatter parameter for diffuse radiation,
+$\beta_0$ is the upscatter parameter for direct radiation,
+$L$ is leaf area index, and $S$ is stem area index.
+
+The equations \eqref{eqn:dif_up}-\eqref{eqn:dif_dn}
+includes the five optical parameters: $G(\mu)$,
+$\bar{\mu}$, $\omega$, $\beta$, and $\beta_0$. The optical
+parameters dependent on wavelength band ($\Lambda$) are
+$\omega$, $\beta$, and $\beta_0$. Next, we describe the
+computation of these optical parameters.
+
+![Image title](../figures/two_stream_approximation.png)
+
+Fig 1. Two-stream approximation of (b) upward diffuse
+radiation, (b) downward diffuse radiation, and (c)
+downward direct radiation.
+
+### Projected leaf and stem area
+
+ELM uses the Ross-Gourdriaan function to compute the
+projection of leaf area in the direction of the solar
+radiation as:
+
+$$
+\begin{equation}
+G(\mu) = \phi_1 + \phi_2 \mu
+\end{equation}
+$$
+
+with $\phi_1 = 0.5 - 0.633 \chi_\ell - 0.366 \chi^2_\ell$
+and $\phi_2 = 0.877 (1 - 2 \phi_2)$. The $\chi_\ell$ is
+the Ross index or leaf/stem orientation index that is
```

Comment:
I think this index is used to represent leaf angle distribution, e.g., vertical, horizontal and spherical. It is better to show the meaning of the specific values of Ross index.